sfts/s3_to_sfts.py

```diff
@@ -161,19 +225,24 @@ def is_processed():
 try:
     xfer_jar = f"{xfer_path}/xfer.jar"
     jna_jar = f"{xfer_path}/jna.jar"
-    print(("trying to call subprocess:\nxfer.jar: "
+    logger.info(("trying to call subprocess:\nxfer.jar: "
            f"{xfer_jar}\njna.jar : {jna_jar}"))
     subprocess.check_call(
```

Comment:
We may want to change this `check_call` to `check_output` https://docs.python.org/3/library/subprocess.html#subprocess.check_output.

Capture the output somehow, like `output = subprocess.check_output(...)`, then shortly after you could log it: `logger.info(output)`. That ought to keep the cron report clean, while preserving any messaging we might want to have for later review.

The exception raised by `check_output` is the same `CalledProcessError` one, so there shouldn't be need to alter the `try ... except` block.

Reply:
You've switched to `check_output`. Resolving this.